src/drivers/displays/tDisplayV1Driver.cpp

```diff
@@ -135,7 +134,6 @@ void tDisplay_ClockScreen(unsigned long mElapsed)
 
   // Push prepared background to screen
   background.pushSprite(0, 0);
-  background.deleteSprite();
```

Comment:
Hi! Not sure if this should be deleted. 
There is a really lack of memory on this devices so must be added